man/passwd.1.xml

```diff
@@ -484,6 +484,12 @@
 	    <para>invalid argument to option</para>
 	  </listitem>
 	</varlistentry>
+	<varlistentry>
+	  <term><replaceable>10</replaceable></term>
+	  <listitem>
+	    <para>an error was returned by pam(3).</para>
```

Comment:
The other bullets don't have a trailing '.'.

Also, the man(3) reference should use formatting:

```
<refentrytitle>pam</refentrytitle><manvolnum>3</manvolnum>
```

Reply:
Oh, thanks for that.

It does make the raw xml a lot harder to read :)

Reply:
I can rewrite these pages to man(7), if you want.  ;-)